source/configuration/read.txt

```diff
@@ -80,58 +80,47 @@ You can configure the following properties to read from MongoDB:
+             ``SamplePartitioner``, see the 
+             :ref:`configuration settings description <conf-samplepartitioner>`.
 
-       - ``MongoShardedPartitioner``
+       - ``ShardedPartitioner``
              A partitioner for sharded clusters. Partitions the
              collection based on the data chunks. Requires read access
-             to the ``config`` database. For configuration settings for
-             the MongoShardedPartitioner, see
-             :ref:`conf-mongoshardedpartitioner`.
-
-       - ``MongoSplitVectorPartitioner``
-             A partitioner for standalone or replica sets. Uses the
-             :dbcommand:`splitVector` command on the standalone or the
-             primary to determine the partitions of the database.
-             Requires privileges to run :dbcommand:`splitVector`
-             command. For configuration settings for the
-             MongoSplitVectorPartitioner, see
-             :ref:`conf-mongosplitvectorpartitioner`.
-
-       - ``MongoPaginateByCountPartitioner``
-             A slow, general purpose partitioner for all deployments.
-             Creates a specific number of partitions. Requires a query
-             for every partition. For configuration settings for the
-             MongoPaginateByCountPartitioner, see
-             :ref:`conf-mongopaginatebycountpartitioner`.
+             to the ``config`` database. To learn more about the 
+             ``ShardedPartitioner``, see the 
+             :ref:`configuration settings description <conf-shardedpartitioner>`.
 
-       - ``MongoPaginateBySizePartitioner``
+       - ``PaginateBySizePartitioner``
              A slow, general purpose partitioner for all deployments.
              Creates partitions based on data size. Requires a query
-             for every partition. For configuration settings for the
-             MongoPaginateBySizePartitioner, see
-             :ref:`conf-mongopaginatebysizepartitioner`.
+             for every partition. To learn more about the 
+             ``PaginateBySizePartitioner``, see the 
+             :ref:`configuration settings description <conf-paginatebysizepartitioner>`.
+
+       - ``PaginateIntoPartitionsPartitioner``
+             A partitioner that creates a specified number of 
+             partitions for a collection. Uses the total number of 
+             documents and the average document size in a collection to 
+             calculate the number of documents to include in each 
+             partition. To learn more about the 
+             ``PaginateIntoPartitionsPartitioner``, see the 
+             :ref:`configuration settings description <conf-paginateintopartitionspartitioner>`.
 
        You can also specify a custom partitioner implementation. For
-       custom implementations of the ``MongoPartitioner`` trait, provide
+       custom implementations of the ``Partitioner`` trait, provide
        the full class name. If you don't provide package names, then
```

Comment:
Remove the `If you don't provide package names` sentence - this is no longer the case.